### DISCUSSION TITLE
Changes in 23.1.0 for SQL procedures

### DISCUSSION THREAD

**Discussion by jvbutera** [dbeaver/dbeaver#20217] [General]
Previously we would test our Oracle application procedures in DBeaver without any issues, which look something like this:
```
DECLARE
  pid varchar(75);
BEGIN
  pid := PBCOM_V4.u_lookup_person(:personId);
  IF (pid IS NOT null) THEN
    PBCOM_V4.person_status(pid);
  END IF;
end;
```
With the release of 23.1  (Version 23.1.0.202306041918), the above now fails with the error:
```
SQL Error [6550] [65000]: ORA-06550: line 8, column 3:
PLS-00103: Encountered the symbol "end-of-file" when expecting one of the following:
   ; <an identifier> <a double-quoted delimited-identifier>
The symbol ";" was substituted for "end-of-file" to continue.
```
If I roll DBeaver back to 23.0.5, this disappears and our code works as it had in the past.

**Comment by fsanfo** [comment]
Same thing here. I rolled back to 23.0.5 and it's ok now.

**Comment by monosabio** [comment]
Same error with version 23.1.1. Returning back to 23.0.5

**Comment by E1izabeth** [comment]
Was fixed in https://github.com/dbeaver/dbeaver/issues/20391
The fix will be available in the next release.

**Reply by jvbutera** [reply]
Thank you @E1izabeth
